api/files/views.py

```diff
@@ -86,14 +85,7 @@ class FileDetail(JSONAPIBaseView, generics.RetrieveUpdateAPIView, FileMixin):
     view_name = 'file-detail'
 
     def get_serializer_class(self):
-        try:
-            target = self.get_target()
-        except (NotFound, Gone, PermissionDenied):
-            return FileDetailSerializer
-        else:
-            if isinstance(target, QuickFilesNode):
-                return QuickFilesDetailSerializer
-            return FileDetailSerializer
+        return FileDetailSerializer
```

Comment:
Ah, I see, with `QuickFile` no longer enabled, the logic ended up being just returning the FileDetailSerializer.